src/SFA.DAS.Commitments.Database/Views/ApprenticeshipSummary.sql

```diff
@@ -95,4 +96,14 @@ SELECT
 		and [Status] = 2 AND [IsResolved] = 0
 		AND [EventStatus] <> 3
 		AND [IsExpired] = 0
+	)
+	LEFT JOIN DataLockStatus dlCourseChangeTriaged on dlCourseChangeTriaged.Id =
```

Comment:
Gonna have to keep an eye on this. I think this is one of the slow queries in Prod. Have you got any bright ideas of how to make it perform better or restructure the way we get the info?

Reply:
Not sure what to do. More specific end-point where we only return what is needed on that page (manage apprentices list)?
TBH I don't get why its a problem now with the relative small amount of data we have.